lib/edit-site-page.php

```diff
@@ -34,7 +34,11 @@ function gutenberg_edit_site_init( $hook ) {
 		$_wp_current_template_content,
 		$_wp_current_template_hierarchy,
 		$_wp_current_template_part_ids;
-	if ( 'gutenberg_page_gutenberg-edit-site' !== $hook ) {
+
+	$allowed_hooks = [ 'gutenberg_page_gutenberg-edit-site', 'toplevel_page_gutenberg-edit-site' ];
+	$allowed_hooks = apply_filters( 'site-editor-hook-paths', $allowed_hooks );
```

Comment:
Just noticed that the CI tests are failing because of this. The hook words need to be separated by underscores.

```suggestion
	$allowed_hooks = apply_filters( 'site_editor_allowed_hooks', $allowed_hooks );
```